jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/MultiPart.java

```diff
@@ -475,7 +475,7 @@ public void fail(Throwable t)
             catch (Throwable x)
             {
                 if (LOG.isDebugEnabled())
-                    LOG.debug("Error closing part {}", this, x);
+                    LOG.atDebug().setCause(x).log("Error closing part {}", this);
```

Comment:
Not necessary here, you are in a catch block already.
The Throwable will exist.